scala-package/examples/src/main/scala/org/apache/mxnet/examples/cnntextclassification/README.md

```diff
@@ -0,0 +1,23 @@
+# CNN Text Classification Example for Scala
+This is the example using Scala type-safe api doing CNN text classification. 
+Currently, I cannot reproduce the same result in the python example here. 
```

Comment:
This example is only for Illustration and not modeled to achieve the best accuracy.